pkg/errs/errno.go

```diff
@@ -228,11 +228,6 @@ var (
 	ErrNetstatTCPSocks = errors.Normalize("TCP socks error", errors.RFCCodeText("PD:netstat:ErrNetstatTCPSocks"))
 )
 
-// hex error
-var (
```

Comment:
I noticed why remove this error?

Reply:
It is weird to wrap the hex error again with `ErrRuleContent`, looks like `[placement:PD:xx]invalid rule, [placement:PD:xx] invalid hex` or so... And it is only used here, in this function, after merging two check functions.

Reply:
But actually, we need to use an individual error type for those error come from the third-party package.

Reply:
Then, what about not using this error in adjustRule? And I will add it back.

Reply:
What about now?

Reply:
IMO, we can keep it as before and return the BadRequest when the error is ErrHexDecoding

Reply:
One more error check..? OK, later I will send a fix.

Reply:
Done